documentation/_puppetdb_nav.html

```diff
@@ -39,6 +39,7 @@ <h2 id="puppetdb">PuppetDB {{ puppetdb_version }}</h2>
       <li><a href="/puppetdb/{{ puppetdb_version }}/scaling_recommendations.html">Scaling Recommendations</a></li>
       <li><a href="/puppetdb/{{ puppetdb_version }}/repl.html">Debugging with Remote REPL</a></li>
       <li><a href="/puppetdb/{{ puppetdb_version }}/load_testing_tool.html">Load Testing</a></li>
+      <li><a href="/puppetdb/{{ puppetdb_version }}/resource_statuses.html">Resource Statuses **PE FEATURE**</a></li>
```

Comment:
+1 to this, don't like the caps. I'd also wonder to what extent "PE" is mainly an internal acronym. My preference would be to spell out Puppet Enterprise in docs, but it's probably a docs question.